cmd/bnsd/app/examples.go

```diff
@@ -13,6 +15,28 @@ import (
 	"github.com/iov-one/weave/x/sigs"
 )
 
+// we fix the private keys here for deterministic output with the same encoding
+// these are not secure at all, but the only point is to check the format,
+// which is easier when everything is reproduceable.
+var (
+	sender = makePrivKey("1234567890")
+	dst    = makePrivKey("F00BA411").PublicKey().Address()
+	guest  = makePrivKey("00CAFE00F00D").PublicKey().Address()
+)
+
+// this repeats the string as long as needed to get 64 digits, then
```

Comment:
Good comment. I think it good to not use `this` to avoid context misunderstanding. First `this` could be replaced with `makePriveKey`. I am not sure what the second `this` is applying to :see_no_evil: